src/Numerics/Mesh/Grids.jl

```diff
@@ -241,6 +244,21 @@ struct DiscontinuousSpectralElementGrid{
     """
     minΔ::MINΔ
 
+    """
+    Map to vertex degrees of freedom
```

Comment:
Map to vertex degrees of freedom: `vertmap[v]` contains the degree of freedom located at vertex `v`.

Reply:
@skandalaCLIMA does this depend on orientation as well?